README.md

```diff
@@ -1,73 +1,83 @@
 # Tarantool
 
-[![Build Status][travis-badge]][travis-url]
-[![Build Status][gitlab-ci-badge]][gitlab-ci-url]
+[![Actions Status][actions-badge]][actions-url]
 [![Code Coverage][coverage-badge]][coverage-url]
 [![Telegram][telegram-badge]][telegram-url]
-[![Slack][slack-badge]][slack-url]
-[![Gitter][gitter-badge]][gitter-url]
-[![Google Groups][groups-badge]][groups-url]
+[![GitHub Discussions][discussions-badge]][discussions-url]
+[![Stack Overflow][stackoverflow-badge]][stackoverflow-url]
 
-https://tarantool.io/en/
-
-Patch submissions and discussion of particular patches https://lists.tarantool.org/mailman/listinfo/tarantool-patches/
-
-General development discussions https://lists.tarantool.org/mailman/listinfo/tarantool-discussions/
+https://www.tarantool.io/en/
 
 Tarantool is an in-memory database and application server.
 
 Key features of the application server:
- * 100% compatible drop-in replacement for Lua 5.1,
-   based on LuaJIT 2.1.
-   Simply use `#!/usr/bin/tarantool` instead of
-   `#!/usr/bin/lua` in your script.
- * full support for Lua modules and a rich set of
-   own modules, including cooperative multitasking,
-   non-blocking I/O, access to external databases, etc
+
+* 100% compatible drop-in replacement for Lua 5.1, based on LuaJIT 2.1.
+  Simply use `#!/usr/bin/tarantool` instead of `#!/usr/bin/lua` in your script.
+* full support for Lua modules and a rich set of [own modules][modules],
+  including cooperative multitasking, non-blocking I/O, access to external
+  databases, [persistent queues][queue], [sharding][vshard], [cluster and
+  application management framework][cartridge], etc
```

Comment:
Not sure about correct punctuation for lists, but it's inconsistent now, this list has a period at the end of the first sentence but not the last, the list below has sentences with no period at the end (and no semicolon). I would start sentences with a capital letter and use a period at the end of each item on the list.

Reply:
I would use this style, if I would do this from scratch. However, sure, there is some inconsistency and there are at least two persons who prefer this formatting. Okay so, changed it to capital letters and periods.